src/navigation/MainNavigator.tsx

```diff
@@ -65,31 +47,20 @@ const MainNavigator: FunctionComponent = () => {
                 headerShown: false,
               }}
             />
-            <Stack.Screen
-              name={Screens.MoreInfo}
-              component={MoreInfo}
-              options={{
-                ...headerScreenOptions,
-                title: t("navigation.more_info"),
-              }}
-            />
-            <Stack.Screen
-              name={Screens.ExposureDetail}
-              component={ExposureDetail}
-              options={{
-                ...headerScreenOptions,
-                title: t("navigation.exposure"),
-              }}
-            />
           </>
         ) : (
           <>
             <Stack.Screen
-              name={Stacks.Onboarding}
+              name={OtherScreens.Welcome}
```

Comment:
Is there any other name we could add here to encapsulate this view? I'm worried about having a catch-all too soon and overuse it 😬

Reply:
Yeah good idea. I'll just call it `WelcomeScreens`.